devmgmtui/src/components/VerifyAuthentication.js

```diff
@@ -6,6 +6,11 @@ export default function(ComposedComponent) {
         componentWillMount() {
             if (!this.props.verify.authenticated) {
                 this.props.history.push('/');
+            } else {
+                const { pathname } = this.props.location;
+                if (pathname === "/") {
```

Comment:
Well, what happens if pathname is not equal to "/" ?

Reply:
The corresponding page would just be displayed